packages/charts/src/charts/donut/donut-chart.component.tsx

```diff
@@ -80,9 +98,13 @@ export const DonutChart: FC<Props> = ({
     dimension: svgDimensions,
     colors: theme.colors,
     type: 'donut',
-    stackTreshold,
+    treshold,
   });
 
+  useEffect(() => {
+    onDataStack(stackedElem);
```

Comment:
Should be condition here that checks if the `onDataStack` is defined as handler is optional. Right now the `plot` throws runtime error.